docs/sdk-and-tools/erdjs/writing-and-running-erdjs-snippets.md

```diff
@@ -3,10 +3,8 @@ id: writing-and-testing-erdjs-interactions
 title: Writing and testing interactions
 ---
 
-Writing and testing interactions
-
 :::important
-This tutorial makes use of `erdjs 10`, which isn't officially released yet (in _beta_ as of 2022-03-24). Therefore, the code examples in this section are subject to change.
+This tutorial makes use of `erdjs 10`.
```

Comment:
name the file as `writing-and-testing-erdjs-interactions.md` ?

Reply:
Actually, I wanted to coin the term `erdjs-snippets` (i.e. Mocha scenarios, runnable in VsCode with a single click): https://github.com/ElrondNetwork/erdjs-snippets.

Is it OK to keep the name?